.projen/deps.json

```diff
@@ -60,11 +60,6 @@
       "name": "jsii-pacmak",
       "type": "build"
     },
-    {
-      "name": "jsii-release",
```

Comment:
Why did this happen? `jsii-release` is still a dependency.

Reply:
It was actually never respected by the release workflow. The publishing jobs do not even checkout the source code and use `npx` to download and run jsii-release. This is also aligned with the decoupling of the "publisher" component in project which is generally applicable and doesn't even require that this will be node.js project (and therefore no package.json even exists). See https://github.com/projen/projen/pull/518